src/impl_constructors.rs

```diff
@@ -11,9 +11,9 @@
 //!
 
 #![allow(clippy::match_wild_err_arm)]
-#[cfg(feature = "std")]
+#[cfg(any(feature = "std", feature = "libm"))]
 use num_traits::Float;
-#[cfg(not(feature = "std"))]
+#[cfg(not(any(feature = "std", feature = "libm")))]
 use num_traits::float::FloatCore as Float;
```

Comment:
These two lines still need to be removed - we can't have a public interface (for example Array::linspace) that changes API in a breaking way (from Float to FloatCore) depending on feature flags